spec/query_spec.cr

```diff
@@ -186,9 +186,8 @@ describe LuckyRecord::Query do
 
   describe "#join methods for associations" do
     it "inner join on belongs to" do
-      insert_a_post
-      post = Post::BaseQuery.new.first
-      insert_a_comment(post.id)
+      post = PostBox.new.save
```

Comment:
I tried to use `PostBox.new.save!` here and it raised:
```
in spec/query_spec.cr:183: instantiating 'PostBox#save!()'

      post = PostBox.new.save!
                         ^~~~~ 
```
But it works fine for `CommentBox.new.post_id(post.id).save!` since `post_id()` returns a form, and under the hood `PostBox.save` does `form.save!`.

Is there are reason why there isn't a `save!` method on the Box itself, because I think this can cause confusion.

Reply:
You are totally right. Could you open an issue about that so usability can be improved?

Reply:
Actually I can do it :) Thanks for bringing it up